components/TransLine.tsx

```diff
@@ -0,0 +1,21 @@
+import { Trans } from 'react-i18next'
+import React from 'react'
+
+export interface TransLineProps {
+  i18nString: string // Expects an i18n string
+  links?: string[] | null
```

Comment:
I did a quick test, and I don't think this actually supports null `links` - I get out `undefined` instead of the string for the key I pass in. Any reason not to hard require the links component?

Reply:
🤔 Interesting. I'll take another look at this!

Reply:
Hmm maybe you could show me what you're seeing? I'm not able to reproduce what you described. 

When I do this:

```react
<TransLine i18nKey="header.alt-image-edd" />
```
I see the full translated string like so (not a super helpful screenshot because it's on white background with no context):
<img width="470" alt="Screen Shot 2021-06-28 at 1 25 26 PM" src="https://user-images.githubusercontent.com/67701/123699588-4f851780-d814-11eb-94f3-084c3d6285ea.png">

I'd prefer not to require the `links` argument because I want to be able to use the `<TransLine>` component for strings that don't have links, but do have `<strong>` or other more complicated translation elements.

Reply:
Ah I was testing via adding another story rather than calling the component 
```
 export const NoLinks = Template.bind({})                                                                                
 TransLine.args = {                                                                                                      
   i18nKey: 'title',                                                                                                     
 } 
```   
![image](https://user-images.githubusercontent.com/6129479/123704770-09d84700-d834-11eb-816d-4ccf86553a30.png)

Reply:
aaaand I spotted my own bug. I didnt rename the variable I was setting args on - it works now!